.travis.yml

```diff
@@ -1,8 +1,11 @@
 language: ruby
 rvm:
-  - 2.5.9
+  - 3.1.0
```

Comment:
Same as above ☝️ 
I've verified that [3.1.2 version is available](https://rubies.travis-ci.org/) in Travis:

Reply:
Perfect thanks, I'm modifying it also.